src/renderer/components/ClaimCollectionRewardDialog/ClaimCollectionRewardDialog.tsx

```
@@ -32,6 +53,49 @@ const ClaimCollectionRewardDialog: React.FC<Props> = (props: Props) => {
     default:
       return <></>
   }
+
+  async function makeTx(avatarAddress: string) {
+    console.log(`makeTx: avatarAddress: ${avatarAddress}`)
+    // create action.
+    const fileName = await tmpName();
+    if (avatarAddress.startsWith("0x"))
+    {
+      avatarAddress = avatarAddress.substr(2);
+    }
+    if (!ipcRenderer.sendSync("claim-monster-collection-reward", avatarAddress, fileName))
+    {
+      console.error(`makeTx: failed action`)
+      return "";
```

How about throwing an error instead of returning an empty string?